amqp_connection.c

```diff
@@ -29,6 +29,7 @@
 #include "php.h"
 #include "php_ini.h"
 #include "ext/standard/info.h"
+#include "ext/standard/md5.h"
```

Comment:
Say, other ext which extract credentials from persistent storage. I agree about md5, idea and approach taken from interbase ext (http://lxr.php.net/xref/PHP_7_0/ext/interbase/interbase.c#921) and it serves as a fool-proof.